org.gnome.Dictionary.yml

```diff
@@ -29,38 +28,13 @@ cleanup:
   - /share/doc
   - /share/gir-1.0
 modules:
-  - name: icons
-    buildsystem: simple
-    build-commands:
-      - install -Dm644 accessories-dictionary-24.png /app/share/icons/hicolor/24x24/apps/accessories-dictionary.png
-      - install -Dm644 accessories-dictionary-32.png /app/share/icons/hicolor/32x32/apps/accessories-dictionary.png
-      - install -Dm644 accessories-dictionary-48.png /app/share/icons/hicolor/48x48/apps/accessories-dictionary.png
-      - install -Dm644 accessories-dictionary-64.png /app/share/icons/hicolor/64x64/apps/accessories-dictionary.png
-      - install -Dm644 accessories-dictionary-128.png /app/share/icons/hicolor/128x128/apps/accessories-dictionary.png
-      - install -Dm644 accessories-dictionary-256.png /app/share/icons/hicolor/256x256/apps/accessories-dictionary.png
-      - install -Dm644 accessories-dictionary-512.png /app/share/icons/hicolor/512x512/apps/accessories-dictionary.png
-    sources:
-      - type: file
-        path: accessories-dictionary-24.png
-      - type: file
-        path: accessories-dictionary-32.png
-      - type: file
-        path: accessories-dictionary-48.png
-      - type: file
-        path: accessories-dictionary-64.png
-      - type: file
-        path: accessories-dictionary-128.png
-      - type: file
-        path: accessories-dictionary-256.png
-      - type: file
-        path: accessories-dictionary-512.png
   - name: gnome-dictionary
     buildsystem: meson
     config-opts:
       - -Dbuild_man=false
     sources:
       - type: archive
-        url: https://download.gnome.org/sources/gnome-dictionary/3.26/gnome-dictionary-3.26.1.tar.xz
-        sha256: 16b8bc248dcf68987826d5e39234b1bb7fd24a2607fcdbf4258fde88f012f300
+        url: https://download.gnome.org/sources/gnome-dictionary/40/gnome-dictionary-40.0.tar.xz 
+        sha256: 2e650c2a4cea8cc4d02d4a583c456cbbc9d1871e918c7dc4de081ded1d830db5
       - type: patch
```

Comment:
You should drop this patch as it was merged upstream, along with the related file. Another change that would be nice to have is migrating from `--socket=x11` to use `--socket=fallback-x11`.

The former, gives the application always access to x11 socket, which might not be needed if the user is running the application under a wayland session. `fallback-x11` only gives it access to the `x11` socket if it can't find the wayland one.